jaws-journey-deploy/orb.yml

```diff
@@ -625,6 +626,12 @@ jobs:
         default: "Journeys CircleCI Orb"
       ssh_key_fingerprint:
         type: string
+      argocd_url:
+        type: string
+        default: ""
```

Comment:
i know this is here for backwards compatibility, but feels a bit dirty.  Do we guard anywhere for the empty string, or does the usage of it cause a clear error?